app/views/users/new.html.erb

```diff
@@ -1 +1,52 @@
-<h1>Welcome to the new user page</h1>
\ No newline at end of file
+<h1>Welcome to the new user page</h1>
+
+<%# <form method='post' action ='/users' accept-charset="UTF-8"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>"> %>
+<%# 
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="email"><br><br>
+
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="username"><br><br>
+
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="password"><br><br>
+
+  <input type="submit">
+
+</form>  %>
+
+<%# <%= form_tag("/users", method: "post") do %> 
+  <%# <%= label_tag(:email, "Email:") %> 
+ <%# <%= text_field_tag(:email) %>
+
+  <%#<%= label_tag(:username, "Username:") %>
+ <%# <%= text_field_tag(:username) %>
+
+  <%#<%= label_tag(:password, "Password:") %>
+ <%# <%= text_field_tag(:password) %>
+
+<%#  <%= submit_tag("Submit") %>
+<%# <% end %>
```

Comment:
1. In line 19 Instead of `"/users"` use `"users_path"`
2. In line 21 instead of `text_field_tag` use `email_field_tag`
3. In line 27 instead of `text_field_tag` use `password_field_tag`